spec/models/user_spec.rb

```diff
@@ -17,5 +17,14 @@
     let(:user) { User.create_by_admin(email: 'lola@o2.pl')}
     it { user.valid_password?('secret').eql? true }
   end
+
+  describe '#to_csv' do
```

Comment:
@synion 
Look at example at link above. We should expect ```UserCsvExport``` to receive new and then generate.